docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -657,6 +658,14 @@ Sets the size of the [SSL buffer](https://nginx.org/en/docs/http/ngx_http_ssl_mo
 _References:_
 [https://www.igvita.com/2013/12/16/optimizing-nginx-tls-time-to-first-byte/](https://www.igvita.com/2013/12/16/optimizing-nginx-tls-time-to-first-byte/)
 
+## ssl-certificate-cache-size
+
+Sets the size of cache that stores parsed SSL certificate objects.
+
+The cache helps reduce memory consumption of SSL context per connection. Increase the value if you have huge amount of certificates.
+
+_**default:**_ is 1000.
```

Comment:
I know this is the LRU cache size, but can we clarify it further? For example, if there are more than 500 certificates, should we increase the cache size?